docs/src/main/asciidoc/hibernate-orm.adoc

```diff
@@ -7,6 +7,7 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 
 include::./attributes.adoc[]
 :config-file: application.properties
+:orm-doc-url-prefix: https://docs.jboss.org/hibernate/orm/5.5/userguide/html_single/Hibernate_User_Guide.html
```

Comment:
Nice! BTW we're using ORM 5.6 now so this should be updated - but let's do that later as it might be useful to backport this one (which would need to be 5.5 in that case)